### ISSUE TITLE
Cannot save language profile

### ISSUE THREAD

**Issue by thibodelanghe** [morpheus65535/bazarr#1968]
**Describe the bug**
Cannot save a new language profile

**To Reproduce**
1. Go to settings > languages
2. Add English to Languages Filter
3. Add new profile with English as language and save 

**Expected behavior**
Saves the profile without an error

**Screenshots**
<img width="836" alt="image" src="https://user-images.githubusercontent.com/38117009/194953178-a93a8e64-0bc8-4073-808b-effc7f39a591.png">

<img width="1234" alt="image" src="https://user-images.githubusercontent.com/38117009/194953226-188a63c2-f6ac-42b2-9904-8a6f25c77e14.png">

**Software (please complete the following information):**
 - Bazarr: 1.1.1
 - Radarr version 4.2.4.6635
 - OS: Linux-5.15.0-48-generic-x86_64-with

**Comment by vBm** [comment]
I've had the exact same issue.
Been told to disable ADS blocker since it can interfere with the payload required for successful save. In my case it was already disabled.
As soon as i tried doing the same change and saving while using Firefox, everything was fine.

TL;DR

Switch browser as a workaround.


Hopefully team will be able to debug it and fix the underlying issue down the road.

**Comment by vBm** [comment]
Can confirm that it is fixed for me when using Chrome on `v1.1.2-beta.20`

Thanks!

**Comment by pedroanisio** [comment]
the same error happens with me. version 1.1.2
Brave 1.44.112  Chromium: 106.0.5249.119

**Comment by morpheus65535** [comment]
> the same error happens with me. version 1.1.2
> Brave 1.44.112 Chromium: 106.0.5249.119

Brave is a shitty browser that isn't supported. Use Chrome or Firefox.

**Comment by rumplin** [comment]
Using 1.1.3-beta.15 and I still have this issue. Incognito browser without ad blockers.
